github-issue-classification/k8s/manifests/download-data.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: "v1"
```

Comment:
If data is saved inside the desired path in the PV, I give a thumbs up for this new manifest.